src/App.jsx

```diff
@@ -14,45 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
+
+function sortGoods(goods, query) {
+  const prepearedGoods = [...goods];
+
+  prepearedGoods.sort((good1, good2) => {
+    switch (query) {
+      case SORT_FIELD_LENGTH:
+        return good1.length - good2.length;
+
+      case SORT_FIELD_NAME:
+        return good1.localeCompare(good2);
+
+      default:
+        return 0;
+    }
+  });
+
+  return prepearedGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reverse, setRevers] = useState('');
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
It is better to make this value a boolean, as it can have only two values (reversed/not)